application-domain/src/main/kotlin/hs/kr/equus/application/domain/application/usecase/UpdateGraduationTypeUseCase.kt

```diff
@@ -34,47 +34,41 @@ class UpdateGraduationTypeUseCase(
             queryApplicationPort.queryApplicationByUserId(userId)
                 ?: throw ApplicationExceptions.ApplicationNotFoundException()
 
+        application.run {
+            if (educationalStatus != request.educationalStatus) {
+                educationalStatus?.let {
+                    deleteOtherCase(
+                        receiptCode!!,
+                        educationalStatus,
+                    )
+                }
+            }
+        }
```

Comment:
```suggestion
       application.educationalStatus?.let {
            if (it != request.educationalStatus) {
                deleteOtherCase(
                    application.receiptCode!!,
                    application.educationalStatus,
                )
            }
        }
```

`!!` 너무 많이 사용된 느낌이 들고 application.run은 굳이 사용하지 않아도 된다는 의견임